code/modules/hydroponics/grown.dm

```diff
@@ -146,15 +146,15 @@
 
 /obj/item/reagent_containers/food/snacks/grown/on_grind()
 	var/nutriment = reagents.get_reagent_amount("nutriment")
-	if(grind_results.len)
+	if(length(grind_results))

```

Comment:
Revert this, more thorough testing earlier revealed `.len` is faster than `length` like we originally thought.